include/mata/nfa/algorithms.hh

```diff
@@ -98,6 +102,22 @@ bool is_universal_naive(const Nfa& aut, const Alphabet& alphabet, Run* cex);
  */
 bool is_universal_antichains(const Nfa& aut, const Alphabet& alphabet, Run* cex);
 
+//the alphabet parameter should be optional? The automaton can have its alphabet too. Run should be optional.
```

Comment:
ok

Reply:
last time we said that automaton would always have an alphabet, so 3 would not apply

Reply:
Note: I am not sure if it is possible to implement such an alphabet. It will need to be investigated further.

And 3. would still apply. We would check for universality for only symbols in the automaton. Therefore, it is up to us whether that seems OK or not.